pkg/ciContext/ciContext.go

```diff
@@ -56,7 +72,7 @@ func Extract() *CIContext {
 }
 
 func isCI() string {
```

Comment:
btw the function name is a bit confusing now. I would expect `isCI` to return boolean but it returns string